packages/grid/_modules_/grid/constants/localeTextConstants.ts

```diff
@@ -41,6 +41,7 @@ export const GRID_DEFAULT_LOCALE_TEXT: GridLocaleText = {
   // Filter panel text
   filterPanelAddFilter: 'Add filter',
   filterPanelDeleteIconLabel: 'Delete',
+  filterPanelLinkOperator: '',
   filterPanelOperators: 'Operators',
```

Comment:
I don't know why it's in plural but it refers to the current value. The same for "Columns".

```suggestion
  filterPanelOperators: 'Operator',
```